File: src/main/java/org/taktik/icure/dto/filter/patient/PatientByHcPartyGenderEducationProfession.java
package org.taktik.icure.dto.filter.patient;

import org.taktik.icure.dto.filter.Filter;
import org.taktik.icure.entities.Patient;
import org.taktik.icure.entities.embed.Gender;

public interface PatientByHcPartyGenderEducationProfession extends Filter<String, Patient> {
    Gender getGender();
    String getEducation();
    String getProfession();
    String getHealthcarePartyId();
}

File: src/main/kotlin/org/taktik/icure/domain/filter/patient/PatientByHcPartyGenderEducationProfession.kt
package org.taktik.icure.domain.filter.patient

import org.taktik.icure.domain.filter.Filter
import org.taktik.icure.entities.Patient
import org.taktik.icure.entities.embed.Gender

interface PatientByHcPartyGenderEducationProfession : Filter<String, Patient> {
    val gender: Gender?
    val education: String?
    val profession: String?
    val healthcarePartyId: String?
}
